### ISSUE TITLE
Adding Windows support on upstream git-repo

### ISSUE THREAD

**Issue by dpursehouse** [esrlabs/git-repo#64]
Hello,

I'm one of the maintainers of git-repo, and I was just made aware of this fork today.

We currently have a [series of patches under review](https://gerrit-review.googlesource.com/#/q/status:open+project:git-repo+branch:master+topic:windows-support) to add support for Windows.

Is this something that you would be interested in helping to review/test, potentially with the aim to contribute some of the work in this fork back upstream?

Thanks.

**Comment by dpursehouse** [comment]
CC @sschuberth @rpaquay

**Comment by fire** [comment]
What is the status of this?

**Comment by dpursehouse** [comment]
@fire the upstream changes that I mentioned in this issue have all been merged, but I can't say for sure whether or not Windows is now 100% supported.

@rpaquay do you have any more fixes planned?

**Comment by rpaquay** [comment]
No, there are no more fixes planned, except maybe support for [SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE](https://msdn.microsoft.com/en-us/library/windows/desktop/aa363866(v=vs.85).aspx) flag when creating symbolic link, but that is a nice to have more than a requirement.
